blog/2020-07-31-temporal-v0.28.0-changelog.md

```diff
@@ -0,0 +1,113 @@
+---
+tags:
+- changelog
+- temporal
+- v1
+posted_on_: 2020-07-31T00:00:00
+id: temporal-v0.28.0-changelog
+title: Temporal v0.28.0 changelog
+author: Cully Wakelin
+author_title: SDE Technical Writer
+author_image_url: https://avatars2.githubusercontent.com/u/34380806?s=400&u=5cd38b5e4416a5d10cdf9ebd386eec1d02f0b067&v=4
+release_version: v0.28.0
+---
+
+<!--truncate-->
+
+Dear Temporal users,
```

Comment:
Too formal, this isn't a letter to the editor :) 

Maybe, "Hey, I'm Cully and I'm...".